MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift

```diff
@@ -11,20 +11,35 @@ import SwiftUI
 class DetailsViewController: UIViewController {
   
     @IBOutlet weak var abilitiesView: UIView!
+    @IBOutlet var generalView: UIView!
     
     var character: CharacterModel?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
+        let yourBackImage = UIImage(named: "back")
+        self.navigationController?.navigationBar.backIndicatorImage = yourBackImage
+        self.navigationController?.navigationBar.backIndicatorTransitionMaskImage = yourBackImage
+        self.navigationController?.navigationBar.backItem?.title = ""
+        
+        let contentView = GeneralView(character: character!)
+        let childView = UIHostingController(rootView: contentView)
+        addChild(childView)
+        childView.view.frame = generalView.bounds
+        generalView.addSubview(childView.view)
+        generalView.addConstrained(subview: childView.view)
+        childView.didMove(toParent: self)
+        
+        /*
```

Comment:
If the code below won't be needed or is outdated, delete it.
If it belongs to another feature, instead of commenting it, adapt it to fit with your code.
Any of the options but never leave commented unnecesary code on the repository

Reply:
Jose the comments was deleted, and right now i pushed the pending fixes